controllers/cloudinit/controlplane_init.go

```diff
@@ -140,6 +140,7 @@ type ControlPlaneInput struct {
 	JoinToken                string
 	ClusterConfiguration     string
 	InitConfiguration        string
+	Version                  string
```

Comment:
You are not using this one in the template. Is this meant to be the snap channel?

Reply:
You are right the version addition here is not needed.